deploy/stacks/trigger_function_stack.py

```diff
@@ -52,6 +54,7 @@ def __init__(
             memory_size=256,
             timeout=Duration.minutes(15),
             environment=env,
+            environment_encryption=env_var_encryption_key,
```

Comment:
Why not create the KMS key in a method within this stack? Are you planning to reuse a single key across all the lambdas? 

Since this is a security requirement (which I obviously missed) I don't think there will be a scenario that we want a TriggerFunction without env_encryption.

Reply:
@noah-paige @petrkalos I added the encryption for both the functions as well. 
With regards to creating the key in each method, this will create too many KMS keys just for encrypting the lambda env variables. I am planning to unify this key to be a single key that will encrypt all lambda functions as well in the next iteration. Currently we have different keys for lambda_api , TriggerFunctions and Cognito.